libraries/DataFlash/LogStructure.h

```diff
@@ -34,6 +36,91 @@ struct PACKED log_Format {
     char labels[64];
 };
 
+struct PACKED log_Unit {
+    LOG_PACKET_HEADER;
+    uint64_t time_us;
+    char type;
+    char unit[64]; // you know, this might be overkill...
```

Comment:
agreed, 64 length for a unit? 16 should be plenty for this but I don't know if it even matters

Reply:
16 is a problem actually on the existing log messages, so this is future proofing :)